doc/ref/copyright.xml

```diff
@@ -66,28 +66,21 @@ make.
 In addition to the general copyright for &GAP; set forth above,
 the following terms apply to the versions of &GAP; for Windows.
 <P/>
-The executable of &GAP; for Windows that we distribute was compiled with
+&GAP; for Windows is packaged with several packages from
+<Package>Cygwin</Package> (<URL>https://cygwin.com</URL>).
+Files from <Package>Cygwin</Package> are provided under their respective
+licenses (included).
+The &GAP; for Windows executables that we distribute are themselves released
+under the terms of the GNU General Public License (GNU GPL);
+either version 2 of the License, or (at your option) any later version.
+These &GAP; executables were compiled with
 the <C>gcc</C> compiler supplied with <Package>Cygwin</Package> installation
-(<URL>https://cygwin.com/</URL>).
 <P/>
 The GNU C compiler is
 <P/>
-<E>Copyright &copyright; 2010 Free Software Foundation, Inc.</E>
+<E>Copyright &copyright; Free Software Foundation, Inc.</E>
 <P/>
-under the terms of the GNU General Public License (GPL).
-<P/>
-The <Package>Cygwin</Package> API library is also covered by the GNU GPL. 
-The executable we provide is linked against this library (and in the process 
-includes GPL'd <Package>Cygwin</Package> glue code). This means that the 
-executable falls under the GPL too, which it does anyhow.
-<P/>
-The <F>cyggcc_s-1.dll</F>, <F>cygncurses-10.dll</F>, <F>cygncursesw-10.dll</F>,
-<F>cygpanel-10.dll</F>, <F>cygpopt-0.dll</F>, <F>cygreadline7.dll</F>,
-<F>cygstart.exe</F>, <F>cygwin1.dll</F>, <F>libW11.dll</F>, <F>mintty.exe</F> 
-and <F>regtool.exe</F> are taken unmodified from the 
-<Package>Cygwin</Package> distribution. They are copyright by RedHat Software
-and released under the GPL. For more information on <Package>Cygwin</Package>,
-see <URL>https://www.cygwin.com</URL>.
+under the terms of the GNU GPL, version 3.
```

Comment:
Wow, I never was aware of this text. Why is this even in there?? I see no reason, can someone elaborate?

Otherwise, let's just remove it all?

Reply:
No idea, I can’t see why either.

Reply:
OK, I traced the origin of this text through history (it wasn't easy; it move through several files in multiple commits; often being added to a new file in one commit, and then removed from the old place in another, later commit). Ultimately, the oldest place introducing it I could find is in a commit from 1998-07-16 with the commit message "Merge" -- it seems a (CVS?) development branch was being merged, but no separated history was recorded. Could be due to a bug in my repository conversion back then, or perhaps the development happened outside CVS, I can't say. But I am pretty sure it means the reasons are lost in the mists of time. So I say: let's just remove it.

Of course we can also wait a bit and see if @alex-konovalov or @ChrisJefferson have anything to add

Reply:
Agree, let's remove it.

Reply:
I don't think it's necessary either -- my suggested fix was a minimal version which I felt handled the same requirement.